libwidgets/View/SearchResults.vala

```diff
@@ -811,17 +817,25 @@ namespace Marlin.View.Chrome {
             }
 
             File? file = null;
-
+            string name = "";
             /* It is important that the next line is not put into an if clause.
              * For reasons unknown, doing so causes a segmentation fault on some systems but not
              * others.  Any changes to the format and content of the accept () function should be
              * carefully checked for stability on a range of systems which differ in architecture,
              * speed and configuration.
              */
-            list.@get (accepted, 3, out file);
 
+
+            list.@get (accepted, 3, out file);
+            list.@get (accepted, 0, out name);
             if (file == null) {
-                Gdk.beep ();
+                if (name == "…") {
```

Comment:
Seems a little bit hacky?

Reply:
I think the reason was that the ellipsis does not need translating so can be used to identify the "More Results ..." row.  I'll try to think of a more elegant solution.